frontend/app/settings/payouts/page.tsx

```
@@ -1,36 +1,36 @@
 "use client";
 
-import { AlertTriangle, Check, Plus, CircleDollarSign } from "lucide-react";
-import { useMutation, useQueryClient, useSuspenseQuery } from "@tanstack/react-query";
-import React, { Fragment, useState } from "react";
-import { z } from "zod";
+
 import MutationButton, { MutationStatusButton } from "@/components/MutationButton";
```

ESLint

[error] Delete `⏎`

(prettier/prettier)